### PR TITLE
Duplicate all protos from udpa tree to xds tree

### PR DESCRIPTION
**Pull request by markdroth**
This is step 1 of the plan from #2.

Question: How do I test something like this to make sure that it's not going to break any callers?

### CODE REVIEW

File: bazel/api_build_system.bzl [hunk 1]
```diff
@@ -146,14 +146,25 @@ def udpa_proto_package(srcs = [], deps = [], has_services = False, visibility =
...
 
-def udpa_go_test(name, **kwargs):
+def xds_go_test(name, **kwargs):
     go_test(
         name = name,
         **kwargs
     )
+
+# Old names for backward compatibility.
+# TODO(roth): Remove these once all callers are migrated to the new names.
```
Comment (htuch) [inline bazel/api_build_system.bzl:162]: This will be removed in this PR?

File: xds/service/orca/v3/orca.proto [hunk 2]
```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
```
Comment (htuch) [inline xds/service/orca/v3/orca.proto:1]: I assume you'll do another pass where you flag in bold letters that the old UDPA version is frozen, stale etc?

**Review by htuch — COMMENTED** [review]
Who do you want to test for breakage?

File: bazel/api_build_system.bzl (hunk 1 above)
Comment (markdroth) [reply bazel/api_build_system.bzl:162]: I was thinking that these build rules might be pulled in by projects that use this repo, so I didn't know if I could remove the old names without breaking anyone.  But if we know that no one is using these rules, then I'm happy to remove them immediately.

File: xds/service/orca/v3/orca.proto (hunk 2 above)
Comment (markdroth) [reply xds/service/orca/v3/orca.proto:1]: Done.

**Review by markdroth — COMMENTED** [review]
I was thinking about tests for projects that depend on this.  I can obviously figure out whether this will break gRPC, but, for example, how do I know if this is going to be okay for Envoy?

**Review by htuch — APPROVED** [review]

**Pull request merged**